docker/single-process/scripts/init

```diff
@@ -5,12 +5,23 @@ export LC_ALL=en_US.UTF-8
 
 cd /app
 
+if ! whoami &> /dev/null; then
```

Comment:
Can this block be done in `setup_env` for the `single-process` and `test` image?

Reply:
Good idea. I'll make the changes.

Reply:
Nice, some of the code is not required by the newer (`single-process` and `test`) images because we never supported the `HUGINN_` prefixed environment variables in them, but it should be fine.